R/crew_launcher_slurm.R

```diff
@@ -49,6 +49,8 @@
 #'   SLURM cluster. `slurm_time_minutes = 60` translates to a line of
 #'   `#SBATCH --time=60` in the SLURM job script. `slurm_time_minutes = NULL`
 #'   omits this line.
+#' @param slurm_partition Character of length 1, name of the slurm partition to
+#'   create workers on.
```

Comment:
Would be great to have a mention of how this translates into the `--partition` flag, as in the above `@param` descriptions. Also how a `NULL` value omits the flag.